index.bs

```diff
@@ -3627,19 +3637,25 @@ Abandoning an Unfinished Technical Report</h5>
 	Any [=technical report=] no longer intended
 	to advance or to be maintained,
 	and that is not being rescinded,
-	<em class="rfc2119">should</em> be [=published=] as a [=Working Group Note=].
+	<em class="rfc2119">should</em> be [=published=]
+	as a [=Discontinued Draft Recommendation=].
 	This can happen if
 	the [=Working Group=] decided
 	to abandon work on the report,
 	or the [=Director=] required the [=Working Group=]
 	to discontinue work on the technical report before completion.
 	If a [=Working Group=] is made to <a href="#GeneralTermination">close</a>,
 	W3C <em class="rfc2119">must</em> [=publish=] any unfinished [=technical report=]
-	on the Recommendation track as [=Working Group Notes=].
+	on the Recommendation track as [=Discontinued Draft Recommendation=].
+
+	Such a document should include in its status section
+	an explanation of why it was discontinued.
 
-	Such a document should be documented as discontinued in its status section,
-	with a <a href="#abandon-draft">link to this section of this document</a>,
-	and accompanied by an explanation of why it was discontinued.
+	A [=Working Group=] <em class="rfc2119">may</em> resume work
+	on a [=technical report=]
+	within the scope of its charter
+	at any time,
+	at the maturity level the specification had before publication as a [=Discontinued Draft Recommendation=].
```

Comment:
This text reproduces what was present before, so this comment shouldn't block merging, but possibly generate a new issue if others agree.

Consider:

WD -> CR -> DDR -> [years pass] -> CR 

It seems potentially risky to permit this without some need to demonstrate wide review of the document again before re-entering CR. For example HR requirements may have changed.

Reply:
two thoughts:
* to resume at a previous maturity higher than WD, we probably should require no textual change (or at least no substantive change)
* Maybe we shouldn't allow resuming as anything other than WD, though we need to be careful about the patent policy implications